packages/salmon-lambda-functions/src/generic/index.ts

```diff
@@ -21,12 +24,13 @@ export async function handleGenericPriceApiProvider (provider: PriceProvider, ev
   const prices = await fetchPrices(env, provider)
   console.log(JSON.stringify({ prices, event }))
 
-  const oraclesManager = OraclesManager.withWhaleClient(env.oceanUrl, env.network, env.privateKey)
+  const oraclesManager =
+    OraclesManager.withWhaleClient(env.oceanUrl, env.network, env.privateKey, new LambdaPrevoutProvider())
 
   try {
-    const txid = await broadcastPrices(oraclesManager, env, prices)
-    if (txid !== undefined) {
-      console.log(`Sent with txid: ${txid}`)
+    const prevout = await broadcastPrices(oraclesManager, env, prices)
+    if (prevout !== undefined) {
+      process.stdout.write(`${JSON.stringify(prevout)}`)
```

Comment:
```suggestion
      process.stdout.write(`${FALLBACK_PREVOUT_PREFIX} ${JSON.stringify(prevout)}`)
```
With specific log group, literally we have partitioned, filterable time series DB :D